src/configs/mod.rs

```diff
@@ -124,6 +125,7 @@ pub struct FullConfig<'a> {
     terraform: terraform::TerraformConfig<'a>,
     time: time::TimeConfig<'a>,
     username: username::UsernameConfig<'a>,
+    v: vlang::VLangConfig<'a>,
```

Comment:
I think this also needs to be `vlang`?

```suggestion
    vlang: vlang::VLangConfig<'a>,
```

Reply:
Hmm, I was trying to follow golang's suite where the module is called golang and config is called go.

Reply:
Ah, makes sense.